blueocean-commons/src/main/java/io/jenkins/blueocean/commons/stapler/export/ExportInterceptor.java

```diff
@@ -32,10 +32,11 @@
      *
      * @param property to get the value from model object
      * @param model object with this property
+     * @param config current configuration to control marshalling / exception-handling behaviour
      * @return the value of the property, if {@link #SKIP} is returned, this property will be skipped
      * @throws IOException if there was a problem with serialization that should prevent
      *         the serialization from proceeding
-     * @see Exported#skipNull()
+     * @see org.kohsuke.stapler.export.Exported#skipNull()
```

Comment:
We do not copy `Exported` as we continue using `org.kohsuke.stapler.export.Exported` in our external interface/model classes. There is not much we can do other than keep this fix locally here till we figure out how excluded packages for javadoc run works with java8.